docs/lake/design.md

```diff
@@ -682,8 +713,8 @@ the relationship between the raw data and the derived data.
 ## Keyless Data
 
 This is TBD.  Data without a key should be accepted some way or another.
-One approach could be to simply assign the "zero-value" as the pool key.
-Or a configured default value could be used.  This would make key-based
+One approach could be to simply assign the "zero-value" as the pool key,
+or a configured default value could be used.  This would make key-based
```

Comment:
Opening with "One approach" really begs for parallel construction in the next clause.
```suggestion
One approach is to simply assign the "zero-value" as the pool key; another
is to use a configured default value.  This would make key-based
```